superdesk/templates/translation_changed.html

```diff
@@ -0,0 +1,5 @@
+{% extends "email_layout.html" %}
+
+{% block content %}
+<p>{{username}} changed the following translated article: <a href="{{link}}">{{link}}</a></p>
```

Comment:
imo here we could use item headline for the link text

Reply:
This was my first intention but then I seen that on user_mentions is used in this way.
I think that the scope was to be able to copy the link.